llama_index/vector_stores/simple.py

```diff
@@ -106,6 +106,7 @@ def delete(self, doc_id: str, **delete_kwargs: Any) -> None:
     def query(
         self,
         query: VectorStoreQuery,
+        index_doc_ids: Set[str],
```

Comment:
Instead of this, we can have a concept of namespace in simple vector store. And vector similarity search could be constrained to docs within a namespace, instead of all documents